Assets/RGLUnityPlugin/Scripts/RadarModels/RadarConfiguration.cs

```diff
@@ -48,16 +48,31 @@ public class RadarConfiguration
         [Tooltip("Maximum elevation angle (up)")]
         [Range(-180.0f, 180.0f)] public float maxElevationAngle;
 
+        [Tooltip("The frequency of the radar (in GHz)")]
+        [Range(0.0f, 100.0f)] public float frequency = 79.0f;
```

Comment:
Zero here may cause issues - e.g. wavelength is calculate dividing speed of light by radar wave frequency. Technically anything above 0 should work, but in fact I would restrict this to radar waves frequency range, at least from the bottom.